### ISSUE TITLE
why set freeze_at = 2？

### ISSUE THREAD

**Issue by hysshy** [facebookresearch/Detectron#859]
Hello, I am reading the detectron, and it do help me a lot. I really appreciate it.
But, there's a problem I don't understand. Why set freeze_at = 2 to freeze the previous （7×7）layer， why stop gradient for the layer.

**Comment by wangqixun** [comment]
+1

**Comment by yushilinGithub** [comment]
that is a transfer learning trick, if your data trained on pre-trained model is very similar to your new data, this is very useful to prevent over-fitting,specifically, this trick can makes your model more general(the other way to make training set bigger) , it is benefit to you accuracy.

**Comment by edugp** [comment]
It might be useful to make this explicit in the docs. I was writing a custom backbone starting with random weights without realizing this was happening. I can open a PR to add it in the "Write models" section in case you find it useful too
